### ISSUE TITLE
Semantic Headings for Cards

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#54]
**Pagina:**
[Rijkshuistijl Community Template](https://rijkshuisstijl-community-templates-git-102644-nl-design-system.vercel.app/developer-overheid/homepage)

**Probleembeschrijving:**
De Cards hebben geen headings waardoor ze lastiger te ‘scannen’ zijn voor screenreadergebruikers.

**Referentiecode:**
`<div class="rhc-card rhc-card--default"><div class="rhc-card__content"><div class="rhc-card__image-container"><img alt="Een open boek" src=" https://developer.overheid.nl/img/boek-opengeslagen.svg" class="utrecht-img utrecht-img--photo rhc-card__image"></div><div class="rhc-card__icon"></div><div class="rhc-card__heading">Kennisbank</div><div class="rhc-card__description">Tutorials, tools, code voorbeelden en meer. Alles wat je nodig hebt om aan de slag te gaan met software van en voor de overheid.</div><div class="rhc-card__metadata"></div></div><div class="rhc-card__footer"><div class="rhc-card__link rhc-card__anchor"><a href="#" class="utrecht-link utrecht-link--html-a rhc-link" aria-label="Kennisbank" title="Kennisbank">Ga naar de kennisbank</a></div></div></div>`

**Gerelateerde WCAG:**
WCAG 1.3.1 Info and relationships

**Referentieafbeelding:**

<img width="1373" alt="A website view where the headings of the components on the page are highlighted. The Card components lack headings." src="https://github.com/user-attachments/assets/85134e1d-0534-47d4-92b9-39a2ad73a6f1" />
